claat/fetch/drive/auth/auth.go

```diff
@@ -215,9 +235,9 @@ func (c *cachedTokenSource) Token() (*oauth2.Token, error) {
 // authorize performs user authorization flow, asking for permissions grant.
 func authorize(conf *oauth2.Config) (*oauth2.Token, error) {
 	aurl := conf.AuthCodeURL("unused", oauth2.AccessTypeOffline)
```

Comment:
nit: `aURL`

Reply:
fixed,PTAL

Reply:
ah, ignore the ptal, I see you pre-approved, thank you!